layout-and-navigation/lesson-1.md

```diff
@@ -50,92 +50,82 @@ app = new kendo.mobile.Application(document.body, { layout: "main-layout" });
+The TabStrip, as the name indicates, provides a tab menu at the base or top of your app (depending on the mobile platform) that allows you to switch between multiple views. Let's add one to the layout we created in the first step. 
 
 <hr data-action="start" />
 
 #### Action
 
-* **a**. To add a Drawer widget, add the following markup to the index.html file before the closing `<body>` tag:
+* **a**. In the index.html file, add initialization markup for the TabStrip in the `data-role="footer"` div of your Layout. The result should look like this:
 ```
-<div data-role="drawer" id="app-drawer">
+<div data-role="layout" data-id="main-layout">
     <div data-role="header">
         <div data-role="navbar">
-            <span data-role="view-title">Menu</span>
+            <span data-role="view-title"></span>
         </div>
     </div>
+    <div data-role="footer">
+        <div data-role="tabstrip">
 
-    <ul data-role="listview">
-        <li><a href="#index" data-icon="home">Home</a></li>
-        <li><a href="views/favorites.html" data-icon="favorites">Favorites</a></li>
-		<li><a href="views/about.html" data-icon="globe">About</a></li>
-    </ul>
-</div>
-```
-
-* **b**. Now, you'll want to remove the TabStrip you added in the last step from your app layout. Go ahead and remove the footer element from your layout so that it looks like this:
-```
-<div data-role="layout" data-id="main-layout">
-    <div data-role="header">
-        <div data-role="navbar">
-            <span data-role="view-title">
-            </span>
         </div>
     </div>
 </div>
 ```
 
-<hr data-action="end" />
+* **b**. Tabs in the TabStrip are simply anchor (`<a>`) elements, so let's add three of those now, inside the `data-role="tabstrip"` div:
+```
+<a href="#index" data-icon="home">Home</a>
+<a href="views/favorites.html" data-icon="favorites">Favorites</a>
+<a href="views/about.html" data-icon="globe">About</a>
+```
 
-Users can activate the Drawer by swiping in from the left by default, but most apps also include a button or link so that users can explicitly invoke this feature. Let's add a button for our drawer next.
+**Tip**: The `data-icon` attribute allows you to specify the image that will accompany a tab or drawer item. You can learn more about icons, including how to create your own, in [this article](http://docs.telerik.com/kendo-ui/mobile/icons).
 
-<hr data-action="start" />
+* **c**. Now, you'll want to remove the Drawer you added in the last step. Delete the button that invokes the drawer (`<a href="#app-drawer" data-rel="drawer" data-role="button" data-align="left" data-icon="details"></a>`) and the full Drawer div(`<div data-role="drawer" id="app-drawer">`) from your index.html file.
```

Comment:
"complete Drawer div" might be a better choice of phrasing. Also, consider placing a space after Drawer div and before the opening bracket.